mscore/musescore.cpp

```diff
@@ -6267,6 +6278,11 @@ int main(int argc, char* av[])
       if (settings.value("mixerVisible", false).toBool())
             mscore->showMixer(true);
 
+      QAction* a = new QAction(getAction("palette-shortcut"));
+      QKeySequence keys = QKeySequence(Qt::Key_T);
```

Comment:
Let's not do that. Shortcuts are supposed to be set from the shortcuts.xml file. Ctrl + Shift + S could be used?